tracer/test/snapshots/OwinWebApi2Tests.NoFF.__path=_api2_delay_0_statusCode=200.verified.txt

```diff
@@ -10,6 +10,7 @@
       aspnet.action: delay,
       aspnet.controller: conventions,
       aspnet.route: api2/{action}/{value},
+      baggage.user.id: doggo,
```

Comment:
Is it expected that all these `OwinWebApi2Tests.*` snapshots now have a `baggage.user.id: doggo` tag?